### PR TITLE
Created Dockerfile

### PR DESCRIPTION
**Pull request by mrblur**
:heavy_plus_sign:  How do I create an admin account?

### CODE REVIEW

**Comment by qarol** [comment]
@mrblur If I remember correctly, You can type in Rails console something like:
`User.create(username: 'username', password: 'password', password_confirmation: 'password', admin: true, confirmed_at: Time.now.utc)`
Let me know if it works.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.3.7
+RUN apt-get -q update && apt-get -y install git make gcc
+WORKDIR /usr/src/app
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+COPY . .
+RUN rake db:schema:load && rake db:migrate
```
Comment (qarol) [inline Dockerfile]: You don't have to run `rake db:migrate`. Let's say "it's already included in first part".

**Review by qarol — COMMENTED** [review]
This file uses development environment so maybe it should be `Dockerfile.dev`?

**Comment by mrblur** [comment]
You forgot to mention password has to be at least 8 characters long. Otherwise console says "User exists" and does nothing.

**Pull request merged**